operator/src/test/java/oracle/kubernetes/operator/helpers/PodHelperTestBase.java

```diff
@@ -462,7 +464,8 @@ public void whenNoPod_retryOnFailure() {
     testSupport.verifyCompletionThrowable(ApiException.class);
   }
 
-  @Test
+  // TBD This test fails - don't know why.
```

Comment:
The unit test fails because you changed the logHome as recorded in an environment variable, but didn't change the test to match.

The test includes (at about line 565):

       .addEnvItem(envItem("LOG_HOME", LOG_HOME + "/" + UID))

but that doesn't appear to be correct any more. Make that match what the log location should be, and the test ought to pass.

Reply:
Unit test updated.

Reply:
Thaks @russgold !

Reply:
Ready for approval @rjeberhard ...